NEW_README.md

```diff
@@ -0,0 +1,25 @@
+# Bevy Assets
+
+A repository of cool Bevy projects. If you would like to share what you're working on, submit a PR! Feel free to create new categories where it makes sense.
+
+This repository can be explored in the [Bevy Asset website](https://bevyengine.org/assets/).
+
+## Asset Format
+
+Assets are described in a toml file:
+
+```toml
+name = "Official Bevy Examples"
+# Description of your asset. It should be less than 100 characters and contains no formatting.
+description = "Learn each Bevy feature from minimal illustrative examples"
+link = "https://github.com/bevyengine/bevy/tree/latest/examples"
+# Optional image to showcase your asset. Should be a png/jpg/gif located next to your toml file.
+image = "official-bevy-examples.png"
+```
```

Comment:
I think using a plugin as an example might be a bit clearer /  is more "asset-ey". "Official Bevy Examples" is a bit of an exceptional case. Given that this will need to expand as we add more fields / will at some point diverge from anything real, maybe we should just fill in dummy values:

"My Cool Plugin"
"A short and sweet description of My Cool Plugin"
"https://github.com/me/my_plugin"
"my_plugin_icon.png"
etc